cockroachcloud/serverless-faqs.md

```diff
@@ -79,11 +79,13 @@ Yes, your free cluster has been automatically migrated to {{ site.data.products.
 
 CockroachDB [automatically collects statistics](../{{site.current_cloud_version}}/cost-based-optimizer.html#table-statistics) in a background process when certain conditions are met (for example, when more than 20% of rows in a table are modified). The statistics are used by the cost-based optimizer to tune statements for higher performance.
 
-When automatic statistics collection starts your cluster may consume RUs above the 100 RUs per second baseline when your workload is otherwise consuming RUs below the baseline. You can [turn off automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but the cost-based optimizer may choose inefficient statement plans as it doesn't have access to the latest statistics.
+When automatic statistics collection starts you may see a spike in RU usage. You can [turn off automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but the cost-based optimizer may choose inefficient statement plans if it doesn't have access to the latest statistics.
```

Comment:
```suggestion
When automatic statistics collection starts, the cluster may experience a spike in RU usage. You can [disable automatic statistics collection](../{{site.current_cloud_version}}/cost-based-optimizer.html#enable-and-disable-automatic-statistics-collection-for-clusters) to avoid these RU bursts, but this may cause the cost-based optimizer to choose inefficient statement plans based on outdated statistics.
```
Maybe not for this PR, but "RU usage" reads strange to me, since RUs are an abstract unit that shows usage. WDYT? Also, is the spike temporary? periodic? Under what circumstances would you _not_ see the increased RUs? I'm just wondering if we can firm up "may" a bit.